src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    private String password;
+    private byte [] salt;
+
+    public User() {
+    }
+
+    public User(String login, String password) {
+        this.email = login;
+        this.password = password;
+    }
```

Comment:
```suggestion
    public User(String email, String password) {
        this.email = email;
        this.password = password;
    }
```
А то как-то не консистентно.

Reply:
Agree, make as Dmitro suggested